### PR TITLE
UIDATIMP-6: Create data import settings page's 3rd pane for Jobs Prof…

### PR DESCRIPTION
**Pull request by VictorSoroka1**
…iles

### CODE REVIEW

File: src/settings/JobProfiles/ViewJobProfile.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { Pane } from '@folio/stripes/components';
+
+// TODO: view component will be developed in UIDATIMP-133
+export default class ViewJobProfile extends Component {
```
Comment (OleksandrYehorovEPAM) [inline src/settings/JobProfiles/ViewJobProfile.js]: Lets avoid default exports
```suggestion
export class ViewJobProfile extends Component {
```

File: src/settings/JobProfiles/resultsFormatter.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      tabIndex="0"
+      role="button"
+      className={sharedCss.selectableCellButton}
+      data-test-select-item
+      onClick={e => e.stopPropagation()}
+    >
+      <Checkbox name={`selected-${record.id}`} />
+    </div>
+  ),
+  name: record => {
```
Comment (OleksandrYehorovEPAM) [inline src/settings/JobProfiles/resultsFormatter.js]: Let's use short syntax as in `selected` formatter above
```suggestion
  name: record => (
    <AppIcon
      size="small"
      app="data-import"
      iconKey="jobProfiles"
      className={sharedCss.baseline}
    >
      {record.name}
    </AppIcon>
  ),
```

File: test/bigtest/interactors/job-profiles.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  collection,
+  clickable,
+  blurrable,
+  value,
+  property,
+} from '@bigtest/interactor';
+
+import MultiColumnListInteractor from '@folio/stripes-components/lib/MultiColumnList/tests/interactor';
+
+const CheckboxInteractor = interactor(class CheckboxInteractorImplementation {
```
Comment (OleksandrYehorovEPAM) [inline test/bigtest/interactors/job-profiles.js]: Let's use less bloated syntax OR consider extending `CheckboxInteractor` from `@folio/stripes-components/lib/Checkbox/tests/interactor`
```suggestion
@interactor class CheckboxInteractor {
  static defaultScope = '[class*=checkbox---]';

  clickLabel = clickable('label');
  blurInput = blurrable('input');
  inputValue = value('input');
  isChecked = property('input', 'checked');

  clickAndBlur() {
    return this
      .clickLabel()
      .blurInput();
  }
}
```

File: src/components/SearchAndSort/SearchAndSort.js [hunk 4]
```diff
@@ -120,6 +120,7 @@ export class SearchAndSort extends Component {
     columnWidths: PropTypes.object,
     resultsFormatter: PropTypes.shape({}),
     defaultSort: PropTypes.string,
+    finishedResourceName: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (VictorSoroka1) [reply src/components/SearchAndSort/SearchAndSort.js]: This is just convenient prop name from the origin S&S component. Let's stick to it.

**Pull request merged**